avm/ptn/sa/content-processing/main.bicep

```diff
@@ -0,0 +1,2183 @@
+              port: 80
+              scheme: 'HTTP'
+            }
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            failureThreshold: 3
+          }
+          {
+            type: 'Startup'
+            httpGet: {
+              path: '/startup'
+              port: 80
+              scheme: 'HTTP'
+            }
+            initialDelaySeconds: 20 // Wait 10s before checking
+            periodSeconds: 5 // Check every 15s
+            failureThreshold: 10 // Restart if it fails 5 times
+          }
+        ]
+      }
+    ]
+    scaleSettings: {
+      maxReplicas: enableScaling ? 3 : 2
+      minReplicas: enableScaling ? 2 : 1
+      rules: [
+        {
+          name: 'http-scaler'
+          http: {
+            metadata: {
+              concurrentRequests: '100'
+            }
+          }
+        }
+      ]
+    }
+    ingressExternal: true
+    activeRevisionsMode: 'Single'
+    ingressTransport: 'auto'
+    //ingressAllowInsecure: true
+    corsPolicy: {
+      allowedOrigins: [
+        '*'
+      ]
+      allowedMethods: [
+        'GET'
+        'POST'
+        'PUT'
+        'DELETE'
+        'OPTIONS'
+      ]
+      allowedHeaders: [
+        'Authorization'
+        'Content-Type'
+        '*'
+      ]
+    }
+  }
+}
+
+//========== Container App Web ========== //
+module avmContainerApp_Web 'br/public:avm/res/app/container-app:0.17.0' = {
+  name: format(resourceNameFormatString, 'caweb-')
+  params: {
+    name: 'ca-${solutionPrefix}-web'
+    location: resourceGroupLocation
+    environmentResourceId: avmContainerAppEnv.outputs.resourceId
+    workloadProfileName: 'Consumption'
+    enableTelemetry: enableTelemetry
+    registries: useLocalBuild == 'localbuild'
+      ? [
+          {
+            server: avmContainerRegistry.outputs.loginServer
+            identity: avmContainerRegistryReader.outputs.principalId
+          }
+        ]
+      : null
+    // registries: useLocalBuild == 'localbuild'
+    //   ? [
+    //       {
+    //         server: publicContainerImageEndpoint
+    //         image: 'contentprocessorweb'
+    //         imageTag: 'latest'
+    //       }
+    //     ]
+    //   : null
+    tags: tags
+    managedIdentities: {
+      systemAssigned: true
+      userAssignedResourceIds: [
+        avmContainerRegistryReader.outputs.resourceId
+      ]
+    }
+    ingressExternal: true
+    activeRevisionsMode: 'Single'
+    ingressTransport: 'auto'
+    //ingressAllowInsecure: true
+    scaleSettings: {
+      maxReplicas: enableScaling ? 3 : 2
+      minReplicas: enableScaling ? 2 : 1
+      rules: [
+        {
+          name: 'http-scaler'
+          http: {
+            metadata: {
+              concurrentRequests: '100'
+            }
+          }
+        }
+      ]
+    }
+    containers: [
+      {
+        name: 'ca-${solutionPrefix}-web'
+        image: (useLocalBuild != 'localbuild')
+          ? '${publicContainerImageEndpoint}/contentprocessorweb:latest'
```

Comment:
Hardcoding latest image is not a good idea, specially if at some point, there is a non backward compatible change that depends on the infrastructure (e.g. depends of a new environment variable).

Recommended to create a parameter with a default value set to a specific version of the image that works for the current - e.g. `param backendContainerImageTag string = 'latest_2025-06-17_704'`

See how we are addressing in [Multi Agent](https://github.com/Azure/bicep-registry-modules/blob/main/avm/ptn/sa/multi-agent-custom-automation-engine/main.bicep#L52)